js/src/features/project/components/Project/ProjectSlugForm.tsx

```diff
@@ -41,25 +41,18 @@ export const ProjectSlugForm: React.FC<Props> = ({
     [project.slug, edit],
   );
 
-  const onFinishFailed = useCallback((errorInfo: any) => {
-    console.log("onFinishFailed:", errorInfo);
-  }, []);
-
   return (
-    <Form
-      form={form}
-      onFinish={onFinish}
-      onFinishFailed={onFinishFailed}
-      initialValues={initialValues}
-    >
+    <Form form={form} onFinish={onFinish} initialValues={initialValues}>
       <Form.Item
         name="slug"
         rules={[
-          { required: true, message: "Please input Slug!" },
+          { required: true, message: "Please input slug" },
           {
```

Comment:
This validation you can unify to `getUrlError`

Reply:
Done